stubs/Markdown/markdown/treeprocessors.pyi

```diff
@@ -1,20 +1,26 @@
 from _typeshed import Incomplete
+from re import Pattern
 from typing import Any
+from xml.etree.ElementTree import Element
 
 from . import util
 
 def build_treeprocessors(md, **kwargs): ...
 def isString(s): ...
 
 class Treeprocessor(util.Processor):
-    def run(self, root) -> Any | None: ...
+    def run(self, root: Element) -> Element | None: ...
 
 class InlineProcessor(Treeprocessor):
     inlinePatterns: Any
     ancestors: Any
     def __init__(self, md) -> None: ...
     stashed_nodes: Any
     parent_map: Any
-    def run(self, tree, ancestors: Incomplete | None = ...): ...
+    def run(self, tree: Element, ancestors: Incomplete | None = ...) -> Element: ...
 
 class PrettifyTreeprocessor(Treeprocessor): ...
+
+class UnescapeTreeprocessor(Treeprocessor):
+    RE: Pattern[str]
```

Comment:
Looks like this should be a `ClassVar` (you'll have to import `ClassVar` from `typing`):

```suggestion
    RE: ClassVar[Pattern[str]]
```